redfish-core/lib/assembly.hpp

```diff
@@ -540,6 +576,49 @@ inline void setAssemblylocationIndicators(
                         "xyz.openbmc_project.adcsensor.service", "replace");
                 }
             }
+
+            // Special handling for LCD and base panel. This is required to
+            // support concurrent maintenance for base and LCD panel.
+            else if (sdbusplus::message::object_path(assembly).filename() ==
+                         "panel0" ||
+                     sdbusplus::message::object_path(assembly).filename() ==
+                         "panel1")
+            {
+                std::string action{};
```

Comment:
Nit: this can be written more succinctly as:

```
std::string action = (readytoremove.value()) ? : ...;

Reply:
We can, but Gunnar is expecting detailed comments describing the behaviour in each case. Adding all those comments on top of one line, IMO will make it more complex to understand.
Also I am not sure if it will benefit from execution point of view.
Kept it so to have better readability. 
Let me know you say on this.